ChegeBryan/politico
# #163545907 Admin can dissolve parties

#### What does this PR do?

Build page for admin, enabling deleting political parties
#### Description of Task to be completed?

- [x] Create a delete party table
- [x] Create dissolve party button

#### How should this be manually tested?
clone this at `https://github.com/ChegeBryan/politico.git` and check out to `ft-admin-dashboard-163545907` jump in the `UI` directory and open `delete_party.html` on your favourite browser.

#### What are the relevant pivotal tracker stories?
[#163545907](https://www.pivotaltracker.com/story/show/163545907)

#### Screenshots
![screencapture-ui-delete_party-html-2019-01-31-16_57_59](https://user-images.githubusercontent.com/40359451/52059536-2268a800-257b-11e9-8af4-6dbb9e879f1e.png)
